### PR TITLE
do-dont-ns-pill

### PR DESCRIPTION
**Pull request by RobTobias123**
updated links and grammar

### CODE REVIEW

File: src/content/docs/components/ns-pill.mdx [hunk 1]
```diff
@@ -36,23 +36,23 @@ Pills can help to create a subtle sense of urgency or to draw a customer’s att
 
 :::do
 - Use solid versions of icons.
-- Use `ns-pill` very sparingly, ideally one or two per journey maximum. Less is more.
-- Use a `yellow` pill to draw attention to an offer or promotional message.
+- Use `<ns-pill>` sparingly, ideally one or two per journey maximum. (Less is more).
+- Use a yellow pill to draw attention to an offer or promotional message.
```
Comment (andij) [inline src/content/docs/components/ns-pill.mdx:40]: This item sounds like it belongs in the Brand guidelines.

File: src/content/docs/components/ns-pill.mdx [hunk 2]
```diff
@@ -36,23 +36,23 @@ Pills can help to create a subtle sense of urgency or to draw a customer’s att
...
 - Use `red` and `lime` pills for dashboard notifications.
```
Comment (andij) [inline src/content/docs/components/ns-pill.mdx:41]: This sounds specific to notifications, and not something that we should necessarily dictate.

File: src/content/docs/components/ns-pill.mdx [hunk 3]
```diff
@@ -36,23 +36,23 @@ Pills can help to create a subtle sense of urgency or to draw a customer’s att
...
 - Keep your copy short and simple for maximum impact.
-- Use `ns-pill` in a component with a direct CTA.
+- Use `<ns-pill>` in a component with a `direct` CTA (Call To Action).
 - Choose colours that are consistent and relevant.
 :::
 
 :::dont
 - Use as validation messages.
-- Create a shouty experience by using `ns-pill` in multiple places. It dilutes the impact.
+- Create a shouty experience by using `<ns-pill>` in multiple places – it dilutes the impact.
```
Comment (andij) [inline src/content/docs/components/ns-pill.mdx:49]: Maybe remove this line as the same sentiment is an item in the Do's

```suggestion
```

File: src/content/docs/components/ns-pill.mdx (hunk 1 above)
Comment (RobTobias123) [reply src/content/docs/components/ns-pill.mdx:40]: agreed - removing

File: src/content/docs/components/ns-pill.mdx (hunk 2 above)
Comment (RobTobias123) [reply src/content/docs/components/ns-pill.mdx:41]: agreed - removing

**Review by andij — APPROVED** [review]

**Pull request merged**